docs/gadget-devel/signing.md

```diff
@@ -0,0 +1,10 @@
+---
+title: 'Signing a Gadget'
+sidebar_position: 720
+description: 'Signing a Gadget'
+---
+
+It's highly recommended to sign your gadget images for security reasons. Signed
+images ensure integrity and authenticity, adding an extra layer of trust. Tools
+like [cosign](https://docs.sigstore.dev/signing/signing_with_containers/) can be
+used for this purpose.
```

Comment:
How to combine/link this with https://github.com/inspektor-gadget/inspektor-gadget/blob/main/docs/getting-started/verify.md#verify-image-based-gadgets?